platform/os_windows.go

```diff
@@ -118,3 +133,64 @@ func KillProcessByName(processName string) {
 	cmd := fmt.Sprintf("taskkill /IM %v /F", processName)
 	ExecuteCommand(cmd)
 }
+
+// Perform platform specific initialization
+func Init(createExtSwitchNetworkType string) error {
+	// Create HNS network to create external switch on windows platform
+	// This allows orchestrators to start CNS which pre-provisions the network so that the
+	// VM network blip / disconnect is avoided when calling cni add for the very first time.
+	return createExtSwitchHnsNetwork(createExtSwitchNetworkType)
+}
```

Comment:
what is the plan to implement the workaround for the cases where VM connectivity never comes back due default route being deleted?

Reply:
No plan. This change only provides a way to create a network to facilitate the ext network creation.